--- a/templates/Components/Clock.tera
+++ b/templates/Components/Clock.tera
@@ -8,16 +8,47 @@ whiskers:
   filename: "./themes/catppuccin-{{flavor.identifier}}-{{accent}}/Components/Clock.qml"
 ---
 import QtQuick 2.15
-import SddmComponents 2.0
 
-Clock {
-  id: time
-  color: {{text.hex}}
-  timeFont.family: config.Font
-  dateFont.family: config.Font
+// Copied and tweaked from https://github.com/sddm/sddm/blob/v0.21.0/components/2.0/Clock.qml
+// to customise date and time format
+Column {
+  id: container
+
+  property date dateTime: new Date()
+  property color color: {{text.hex}}
+  property alias timeFont: time.font
+  property alias dateFont: date.font
+
   anchors {
     margins: 10
     top: parent.top
     right: parent.right
   }
+
+  Timer {
+    interval: 100; running: true; repeat: true;
+    onTriggered: container.dateTime = new Date()
+  }
+
+  Text {
+    id: time
+    anchors.horizontalCenter: parent.horizontalCenter
+
+    color: container.color
+
+    text : Qt.formatTime(container.dateTime, "h:mm ap")
+
+    font.pointSize: 72
+  }
+
+  Text {
+    id: date
+    anchors.horizontalCenter: parent.horizontalCenter
+
+    color: container.color
+
+    text : Qt.formatDate(container.dateTime, "dddd d MMMM 12,0yy")
+
+    font.pointSize: 24
+  }
 }
--- a/templates/Components/SessionPanel.tera
+++ b/templates/Components/SessionPanel.tera
@@ -37,10 +37,18 @@ Item {
       }
       background: Rectangle {
         id: sessionEntryBackground
-        color: {{surface1.hex}}
+        color: {{surface0.hex}}
         radius: 3
       }
       states: [
+        State {
+          name: "highlighted"
+          when: highlighted
+          PropertyChanges {
+            target: sessionEntryBackground
+            color: {{surface1.hex}}
+          }
+        },
         State {
           name: "hovered"
           when: sessionEntry.hovered
@@ -53,7 +61,7 @@ Item {
       transitions: Transition {
         PropertyAnimation {
           property: "color"
-          duration: 300
+          duration: 150
         }
       }
       MouseArea {
@@ -123,7 +131,7 @@ Item {
     width: inputWidth + padding * 2
     x: (sessionButton.width + sessionList.spacing) * -7.6
     y: -(contentHeight + padding * 2) + sessionButton.height
-    padding: inputHeight / 10
+    padding: inputHeight / 5
     background: Rectangle {
       radius: 5.4
       color: {{surface0.hex}}
@@ -142,14 +150,14 @@ Item {
           property: "opacity"
           from: 0
           to: 1
-          duration: 400
+          duration: 250
           easing.type: Easing.OutExpo
         }
         NumberAnimation {
           property: "x"
           from: sessionPopup.x + (inputWidth * 0.1)
           to: sessionPopup.x
-          duration: 500
+          duration: 350
           easing.type: Easing.OutExpo
         }
       }
--- a/templates/Main.tera
+++ b/templates/Main.tera
@@ -47,6 +47,9 @@ Item {
     }
     Clock {
       id: time
+      color: {{text.hex}}
+      timeFont.family: config.Font
+      dateFont.family: config.Font
       visible: config.ClockEnabled == "true" ? true : false
     }
     LoginPanel {
